FILE_HEADER

```diff
@@ -1,4 +1,4 @@
-// Copyright {20\d{2}}-{20\d{2}} Parity Technologies (UK) Ltd.
+// Copyright {20\d{2}}-{20\d{2}} Use Ink (UK) Ltd.
```

Comment:
Is `Use Ink` a company registered in the United Kingdom?

Reply:
Yes, cc @evilrobot-01 @peterwht.

Reply:
Correct! 

USE INK LTD
Company number 15664482